packages/app-store/zohocrm/api/add.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
[We prefer `defaultHandler` for api routes. ](https://handbook.cal.com/engineering/best-practices/prefer-defaulthandler-for-simple-api-handlers)
See how in this app https://github.com/calcom/cal.com/blob/4be2eee2da2f39395ba7b448af168606ee825c99/packages/app-store/around/api/add.ts#L6

We are yet to clean it up in existing apps.